package.json

```diff
@@ -70,6 +70,7 @@
     "knox": "^0.9.2",
     "lodash": "^4.17.4",
     "lodash.clonedeep": "^4.3.1",
+    "lodash.set": "^4.3.2",
```

Comment:
Ah do you mind removing this along with `lodash.clonedeep` since we have the full lib installed?